shared/js/utils.js

```diff
@@ -812,6 +814,32 @@ if (inChrome) {
     return node;
   }
 
+  function getPageMetadata(url) {
+    return new Promise((resolve, reject) => {
+      let result = { url };
+      let request = new XMLHttpRequest();
+      request.open("GET",
+        `https://api.embedly.com/1/extract?url=${encodeURIComponent(url)}&key=${encodeURIComponent(API_KEY)}`);
+      console.info(request, `https://api.embedly.com/1/extract?url=${encodeURIComponent(url)}&key=${encodeURIComponent(API_KEY)}`);
```

Comment:
is the `console.info` needed here? if it is, can we store the values to avoid double calling?